src/modal/modal.css

```diff
@@ -0,0 +1,137 @@
+:global {
```

Comment:
Are these bootstrap css rules? I think we could trim out the vendor specific prefixes because our css is getting processed by postcss before it hits the lib directory anyways.